src/OrbitGl/TimerTrack.cpp

```diff
@@ -59,8 +59,7 @@ float TimerTrack::GetYFromTimer(const TimerInfo& timer_info) const {
 }
 
 float TimerTrack::GetYFromDepth(uint32_t depth) const {
-  return pos_[1] - GetHeaderHeight() - layout_->GetSpaceBetweenTracksAndThread() -
-         box_height_ * static_cast<float>(depth + 1);
+  return pos_[1] - GetHeaderHeight() - box_height_ * static_cast<float>(depth + 1);
```

Comment:
See my related comment above, why the change in the usage of `layout_->GetSpaceBetweenTracksAndThread()`?

Reply:
Here it is a bit simpler to explain: TimerTrack should not know at all about the possibility of event/thread bars above the timer pane. Only if the track itself actually has an event bar, it should handle this.

As above, it will become obsolete by the refactorings anyways.